test/Feature/BFSSearcher.c

```diff
@@ -0,0 +1,22 @@
+// RUN: %llvmgcc %s -emit-llvm -g -c -o %t1.bc
+// RUN: rm -rf %t.klee-out
+// RUN: %klee --output-dir=%t.klee-out --max-time=3 --search=bfs %t1.bc 2>%t2.log
```

Comment:
Instead of spending 3s for this test, I propose to run it for a small number of instructions.